spec/yaml/swiftnav/sbp/acquisition.yaml

```diff
@@ -246,3 +246,5 @@ definitions:
             type: array
             fill: AcqSvProfileDep
             desc: SV profiles during acquisition time
+
```

Comment:
This is just to force the check generated artefacts stage to run in CI. I'll remove it before merging